### PR TITLE
Obpih-3244, 3245:  Add GL Account Type and GL Account

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -1197,6 +1197,8 @@ openboxes {
                         [label: "documents.label", defaultLabel: "Documents", href: "/${appName}/document/list"],
                         [label: "documentTypes.label", defaultLabel: "Document Types", href: "/${appName}/documentType/list"],
                         [label: "eventTypes.label", defaultLabel: "Event Types", href: "/${appName}/eventType/list"],
+                        [label: "glAccountType.label", defaultLabel: "GL Account Type", href: "/${appName}/glAccountType/list", requiredRoles: [RoleType.ROLE_ADMIN, RoleType.ROLE_SUPERUSER]],
```
Comment (jmiranda) [inline grails-app/conf/Config.groovy:1200]: This isn't a big deal, but can you figure out a way to implement the requiredRoles using the lowest allowed role. I know this is implemented somewhere using getEffectiveRoles() or getHighestRole(). We basically just want to include the lowest common denominator here. It's possible that the taglib that is used to check if a user has a role in the megamenu does this same logic. And if that's the case just leave it as-is and we'll refactor later.

File: grails-app/controllers/org/pih/warehouse/core/GlAccountController.groovy [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.core
+
+class GlAccountController {
+
+    def scaffold = GlAccount
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/core/GlAccountController.groovy:14]: For what it's worth, the scaffold property allows you to exclude the actions below, unless you're overriding them.

**Review by jmiranda — APPROVED** [review]
Looks good to me. If @DominikaHatala approves, feel free to merge.

**Review by DominikaHatala — APPROVED** [review]

File: grails-app/controllers/org/pih/warehouse/core/GlAccountController.groovy (hunk 2 above)
Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/core/GlAccountController.groovy:14]: I'll remove it in next pr

File: grails-app/conf/Config.groovy (hunk 1 above)
Comment (awalkowiak) [reply grails-app/conf/Config.groovy:1200]: I'll create a ticket for this

**Pull request merged**